bessctl/bessctl

```diff
@@ -141,14 +142,15 @@ def main():
         line_buf = []
 
         for arg in sys.argv[1:]:
+            arg = arg.decode()
```

Comment:
Sure, but you'll have to not do `.decode()` if it's py3k (either that, or trap `AttributeError` as there is no decode for str objects).